src/commands/general/ping.ts

```diff
@@ -0,0 +1,10 @@
+import { CommandInterface } from "../../interfaces/CommandInterface";
+
+export const ping: CommandInterface = {
```

Comment:
lgtm, ~~but can we also have a pong command~~ (for documentation purpose, ofcourse)

Reply:
What would it do and why?

Reply:
nvm 😅  (ref: symmetric documentation, hf)